docs/DesignTokens/Taxonomy.stories.mdx

```diff
@@ -0,0 +1,138 @@
+import { Meta } from "@storybook/addon-docs";
+
+<Meta title="Design Tokens Taxonomy" />
+
+# 🚧 WIP
+
+## Design system namespace
+
+All tokens start with the namespace for the design system, `cpd`.
+
+## Token tiers
+
+Our design tokens are conceptually divided into three tiers:
+
+- System
+- Reference
+- Components
+
+The top level is global and not aware of context. Subsequent tiers are context-aware, each one being more precise.
+
+System tokens describe plain values, while reference tokens describe their purpose in specific contexts. Component tokens describe the intended use in the context of a specific component.
+
+With three kinds of tokens, we can update design decisions globally or apply a change to a single component.
+
+### System
+
+Namespace: `sys`
+🔴 TODO: Consider removing namespace.
+
+Use for global, immutable literal values.
+
+### Reference
+
+Namespace: `ref`
+🔴 TODO: Consider removing namespace.
+
+Reference tokens are semantic aliases that map system tokens to specific contexts. They represent the choices the design system team made in regards to when to use which token.
+
+It allows us to granularly target and control the mapping of system tokens to specific contexts, decoupling it from other contexts that happen to consume the same system token.
+
+### Component
+
+Namespace: `cmp`
+🔴 TODO: Consider removing namespace.
+
+Semantic aliases for mapping tokens to component-specific contexts.
+
+## Token taxonomy
+
+### Namespace
+
+- Design system
+  - `cpd`
+- Theme — Define themes by extending and overriding default tokens. Useful for brands, sub-brands, white-labeling, and color schemes.
+  - `ees`
```

Comment:
I don't think this matches the reality of what we started building. 

There are no namespaces for theme, we have file-specific override that trickle down the the theme semantics. I believe we should remove that

Reply:
Removed. 👍